include/crud-posts.php

```diff
@@ -324,4 +325,32 @@ public function create_media_translation( $post_id, $lang ) {
 		do_action( 'pll_translate_media', $post_id, $tr_id, $lang->slug );
 		return $tr_id;
 	}
+
+	/**
+	 * Forces tags translation when a post is updated, due to `tags_input` paramter being removed in `wp_update_post()`.
+	 *
+	 * @since 3.4.4
+	 *
+	 * @param int     $post_id      Post ID, unused.
+	 * @param WP_Post $post_after   Post object following the update.
+	 * @param WP_Post $post_before  Post object before the update.
+	 * @return void
+	 */
+	public function force_tags_translation( $post_id, $post_after, $post_before ) {
+		remove_action( 'post_updated', array( $this, 'force_tags_translation' ) );
```

Comment:
Please don't do that. 
Due to [a bug in WP](https://core.trac.wordpress.org/ticket/17817) that I'm quite sure I still encountered the issue after it was supposedly fixed, you can't remove an action in itself. All actions after yours will never be executed. 
Using a static variable allows you to achieve the same goal without any risk. See for example https://github.com/polylang/polylang/blob/3.4.4/include/filters-sanitization.php#L67-L78
It may even simplify your code.

Reply:
For the reference: https://core.trac.wordpress.org/ticket/40393

Reply:
Wow great to know, thanks!
Fixed in 4e8bfe69.